examples/src/components/CodePreview.js

```diff
@@ -1,12 +1,10 @@
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import CopyButton from './CopyButton'
-import 'prismjs'
```

Comment:
maybe we can't remove this?

Reply:
You're right, I believe is the last commit which is causing this issue, going to revert it.

Reply:
Looks like it needs additional configuration after the `prismjs` removal to keep the font the same, but I don't have time on it now, the pr #1110  is made to revert this. I'll come back later for it, sorry for the bug.